ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -268,44 +284,80 @@ function CollectionForm({
                             </FlexItem>
                         </Flex>
                         <Divider component="div" />
-                        <Flex
-                            className="pf-u-background-color-200 pf-u-p-lg"
-                            spaceItems={{ default: 'spaceItemsMd' }}
-                            direction={{ default: 'column' }}
-                        >
-                            <Card>
-                                <CardBody>
+                        <Form>
+                            <Flex
+                                className="pf-u-background-color-200 pf-u-p-lg"
+                                spaceItems={{ default: 'spaceItemsMd' }}
+                                direction={{ default: 'column' }}
+                            >
+                                <div className="pf-u-background-color-100 pf-u-p-lg">
                                     <Title headingLevel="h2">Collection details</Title>
-                                </CardBody>
-                            </Card>
-                            <Card>
-                                <CardBody>
-                                    <Title headingLevel="h2">Add new collection rules</Title>
-                                    <RuleSelector />
-                                    <RuleSelector />
-                                    <RuleSelector />
-                                </CardBody>
-                            </Card>
-                            <Card>
-                                <CardBody>
+                                </div>
+
+                                <Flex
+                                    className="pf-u-background-color-100 pf-u-p-lg"
+                                    direction={{ default: 'column' }}
+                                    spaceItems={{ default: 'spaceItemsMd' }}
+                                >
+                                    <Title className="pf-u-mb-xs" headingLevel="h2">
+                                        Add new collection rules
+                                    </Title>
+                                    <p>
+                                        Select deployments via rules. You can use regular
+                                        expressions (RE2 syntax).
+                                    </p>
+                                    <Divider className="pf-u-mb-lg" component="div" />
+                                    <RuleSelector
+                                        entityType="Deployment"
+                                        scopedResourceSelector={values.selectorRules.Deployment}
+                                        handleChange={onRuleSelectorChange}
```

Comment:
Possible follow up to find a more consistent naming convention:
* `selectorRules` is property name in the response, correct?
* `scopedResourceSelect` prop name
* `onRuleSelectorChange` callback function name

Reply:
Hmm good catch here, I knew something felt off. What I was calling `selectorRules` in the FE type property is probably more similar to `resourceSelectors` in the BE response. The difference is the FE type is more restrictive, and pulls the `fieldName` up to the `resourceSelector` level when the BE places it on the individual `selectorRules`.

I'll update this to be:

- `resourceSelectors` in the BE response & FE data object
- `scopedResourceSelector` prop name
- `onResourceSelectorChange` callback function name